src/dispatch/plugins/dispatch_slack/incident/interactive.py

```diff
@@ -1500,7 +1500,7 @@ def handle_report_tactical_command(
 
     incident = incident_service.get(db_session=db_session, incident_id=context["subject"].id)
     if incident.tasks:
-        actions = "\n".join(
+        actions += "\nOutstanding Incident Tasks:\n".join(
```

Comment:
I'd suggest we add another `\n` to have better separation between what was in the most recent tactical report and the list of outstanding tasks
```suggestion
        actions += "\n\nOutstanding Incident Tasks:\n".join(
```